app/views/spree/api/affirm/payload.json.jbuilder

```diff
@@ -0,0 +1,77 @@
+json.config do
```

Comment:
This commit should be in the same commits that define the endpoint that renders the view, and provides the tests for the endpoint.

Did we want to use JBuilder here? If we do, we enforce that dependency on everyone who uses the extension. If we went with a basic model based approach, either with AMS or a regular class, people can generate this json if they decide to use the extension in a way we have not originally accounted for.